### PR TITLE
Add "-disableluarefresh" to gmod startup

### PR DESCRIPTION
**Pull request by Boy132**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

### CODE REVIEW

**Comment by parkervcp** [comment]
@Boy132 can you explain why this would be needed? If it's useful to you but it breaks others server than I will say no.

**Comment by Boy132** [comment]
> @Boy132 can you explain why this would be needed? If it's useful to you but it breaks others server than I will say no.

This is pretty much a default setting for every server to prevent reloading lua when uploading stuff.
Also not disabling it will most likely break stuff - especially when using darkrp as gamemode.

**Comment by parkervcp** [comment]
Well that at least makes sense, but this should a toggle and not a default that can't be changed.

File: stock-eggs/source-engine/egg-garrys-mod.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "author": "support@pterodactyl.io",
     "description": "Garrys Mod, is a sandbox physics game created by Garry Newman, and developed by his company, Facepunch Studios.",
     "image": "quay.io\/pterodactyl\/core:source",
-    "startup": ".\/srcds_run -game garrysmod -console -port {{SERVER_PORT}} +ip 0.0.0.0 +host_workshop_collection {{WORKSHOP_ID}} +map {{SRCDS_MAP}} +gamemode {{GAMEMODE}} -strictportbind -norestart +sv_setsteamaccount {{STEAM_ACC}} +maxplayers {{MAX_PLAYERS}}  -tickrate {{TICKRATE}}",
+    "startup": ".\/srcds_run -game garrysmod -console -port {{SERVER_PORT}} +ip 0.0.0.0 +host_workshop_collection {{WORKSHOP_ID}} +map {{SRCDS_MAP}} +gamemode {{GAMEMODE}} -strictportbind -norestart +sv_setsteamaccount {{STEAM_ACC}} +maxplayers {{MAX_PLAYERS}}  -tickrate {{TICKRATE}} -disableluarefresh",
```
Comment (Software-Noob) [inline stock-eggs/source-engine/egg-garrys-mod.json:11]: You should create a new egg variable to enable/disable this option, for example a Boolean or a string comparison.

`$( [ "$DISABLE_LUA" == "0" ] || printf %s '-disableluarefresh' )` as example

**Pull request closed without merging**